### DISCUSSION TITLE
string-ts logo

### DISCUSSION THREAD

**Discussion by gustavoguichard** [gustavoguichard/string-ts#101] [Polls]
Guys, the good folks from numi.tech are designing a logo for our project. They came up with these 3 options as an early draft.
Let's vote!

![String TS 1](https://github.com/gustavoguichard/string-ts/assets/566971/11b76d65-6286-48c5-84bb-26b477b46bb2)

1 - Its a string forming the letters T and S
2 - I was able to create a monogram with both letters inside a cube
3 - Its a more mature logo with both letters intertwined like strings

@p9f , @jly36963 , @mjuksel ?

**Comment by mjuksel** [comment]
I really like the idea behind the string on Numero Uno!

**Reply by jly36963** [reply]
@mjuksel same, I thought this one conveys `string` the best

**Reply by gustavoguichard** [reply]
I'm between 1 and 3.. the third looks like a string around a needle and would make it for a great favicon IMO

**Reply by garusis** [reply]
I like #3 because my brain relates it to an inflexible string like... Strong typed string 😁.

**Reply by gustavoguichard** [reply]
@garusis , 3 does look solid indeed!
It'd be easy to make a logo for `make-service` as well, just making that T look like an M lol

**Comment by gustavoguichard** [comment]
Folks, we have a logo now! Check it out in the README <3
